[a11y] Inline error a11y docs (#1073)
Shopify/polaris-react: src/components/InlineError/README.md

## Related components

- To create a list of exceptions that describe a resource, [use the exception list component](/components/lists-and-tables/exception-list)

---

## Accessibility

<!-- content-for: android -->

See Material Design and development documentation about accessibility for Android:

- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)

<!-- /content-for -->

<!-- content-for: ios -->

See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:

- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)

<!-- /content-for -->

<!-- content-for: web -->

> - Use the required `fieldID` prop to give the inline error a unique `id`. This ties the error to a form field using `aria-describedby` so it is conveyed to screen reader users.

Reviewer: ```suggestion
- Use the required `fieldID` prop to give the inline error a unique `id`. This ties the error to a form field using `aria-describedby` so that it's conveyed to screen reader users.
```

> - Use the required `message` prop to provide the text of the error.

Reviewer: - Is it helpful to say "text that describes the error" here instead of "text of the error"?

Author: Yup, works for me.

> - Use the required `message` prop to provide the text of the error.

Author: ```suggestion
- Use the required `message` prop to provide the text that describes the error.
```
